src/atomvm_pico_flash_provider.erl

```diff
@@ -210,9 +209,22 @@ do_flash(ProjectApps, PicoPath, ResetPort) ->
             case ResetStatus of
                 "" ->
                     ok;
-                _Any ->
-                    rebar_api:error("Reset ~s failed. Is tty console attached?", [ResetPort]),
-                    rebar_api:abort()
+                Error ->
+                    rebar_api:warn("Warning: ~s~nFor faster flashing remember to disconnect serial monitor first.", [Error]),
+                    Reboot = lists:join(" ", [
+                        "picotool", "reboot", "-f"
+                    ]),
+                    rebar_api:warn("Disconnecting serial monitor with: `~s'", [Reboot]),
+                    RebootStatus = os:cmd(Reboot),
+                    case RebootStatus of
+                        "The device was asked to reboot into application mode.\n" ->
```

Comment:
Can we add a log line (info?) that says we are going to reset after 5 seconds?

Reply:
We could, but the device should have already been reset into `BOOTSEL` by the previous command, this is a fallback if that first reset failed because the console was connected. That 5 second delay is to let that reset settle so it can be put into `BOOTSEL` mode.

Reply:
Naming that command Reset internally might be a bad choice, it is more of a mode change. The Reboot command is closer to a soft reset.

Reply:
I also forgot I need to add an os:find_executable(picotool)
to make sure it's in the users path, and tell them to close the serial monitor before flashing, or install the optional `picotool`, for some reason I was thinking it was listed as required, rather than just recommended.

Reply:
Also, this should be tested with `minicom`, it works flawlessly with `screen` on top of the PR I will be submitting to the AtomVM repo shortly.

Reply:
> I also forgot I need to add an os:find_executable(picotool) to make sure it's in the users path, and tell them to close the serial monitor before flashing, or install the optional `picotool`, for some reason I was thinking it was listed as required, rather than just recommended. I will add a note about that to the documentation here.

Done.

Reply:
> Naming that command Reset internally might be a bad choice, it is more of a mode change. The Reboot command is closer to a soft reset.

I changed the internal command names to hopefully make them more clear.

Reply:
> Can we add a log line (info?) that says we are going to reset after 5 seconds?

I added a 5 second warning before the actual reset, and a comment about the second timer:sleep/1 so someone doesn't remove it thinking it was a mistake.